wpilibNewCommands/src/main/native/include/frc2/command/Commands.h

```diff
@@ -167,35 +167,91 @@ template <typename Key>
 
 // Command Groups
 
+/**
+ * Create a vector of commands.
+ */
+template <typename... Args>
+std::vector<CommandPtr> vec(Args&&... args) {
+  std::vector<CommandPtr> data;
+  data.reserve(sizeof...(Args));
+  (data.emplace_back(std::forward<Args>(args)), ...);
+  return data;
+}
```

Comment:
Is this an implementation detail?  If so, it should be in a `namespace impl { ... }` block

Reply:
There's a fair use case of wanting to create a `vector<CommandPtr>`, it came up multiple times in unit tests I wrote in a yet-to-be-pushed branch.
I'm not opposed to moving it to an impl namespace, other than that it'll be very nested (`frc2::cmd::impl` is rather verbose). Should I move it?

Reply:
I guess I'd just want a better name for it, since it doesn't follow our function naming convention.  `MakeVector()`?

Reply:
Users can use it directly to make a `vector<CommandPtr>`, but I doubt it's likely users will want to do that often.

Reply:
Moved and renamed.